src/cascadia/TerminalApp/App.cpp

```diff
@@ -624,12 +279,14 @@ namespace winrt::TerminalApp::implementation
         //  - display a loading error
         _settingsLoadedResult = _TryLoadSettings(false);
 
+        auto terminalPage = _root.as<TerminalPage>();
+
         if (FAILED(_settingsLoadedResult))
         {
-            _root.Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [this]() {
+            _root.Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [this, terminalPage]() {

```

Comment:
Do we necessarily need to capture terminalPage here? Can't we just get it using the `this` that's already captured?